README.md

```diff
@@ -1,63 +1,41 @@
+Effortlessly (Delete what's not possible such as upload) optimize, transform, upload and manage your cloud's assets.
 
-We maintain difference branches for different versions. Cloudinary's latest Angular SDK can be found [here](https://github.com/cloudinary/cloudinary_angular/tree/angular-5.x)
 
-[Cloudinary](https://cloudinary.com) is a cloud service that offers a solution to a web application's entire image management pipeline. 
+#### Note
+This Readme provides basic installation and usage information. 
+For the complete documentation, see the [Angular SDK Guide](https://cloudinary.com/documentation/angular_integration).
 
-Easily upload images to the cloud. Automatically perform smart image resizing, cropping and conversion without installing any complex software. Integrate Facebook or Twitter profile image extraction in a snap, in any dimension and style to match your website’s graphics requirements. 
-Images are seamlessly delivered through a fast CDN, and much much more. 
 
-Cloudinary offers comprehensive APIs and administration capabilities and is easy to integrate with any web application, existing or new.
+## Table of Contents
+- [Key Features](#key-features)
+- [Version Support](#Version-Support)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Transform and Optimize Assets](#Transform-and-Optimize-Assets)
+    - [Generate Image and HTML Tags](#Generate-Image-and-Video-HTML-Tags)
+    
 
-Cloudinary provides URL and HTTP based APIs that can be easily integrated with any Web development framework. 
+## Key Features
+- [Transform](https://cloudinary.com/documentation/angular1_video_manipulation#video_transformation_examples) and [optimize](https://cloudinary.com/documentation/angular1_image_manipulation#image_optimizations) assets.
+- Generate [image](https://cloudinary.com/documentation/angular1_image_manipulation#deliver_and_transform_images) and [video](https://cloudinary.com/documentation/angular1_video_manipulation#video_element) tags.
 
-For Angular, Cloudinary provides an SDK for simplifying the integration even further. 
-The SDK serves as a layer on top of one of [Cloudinary's Javascript library](https://github.com/cloudinary/cloudinary_js).
 
-
-## Live examples ##################################################################
-Start experimenting right away with one of these live examples:
-
-* [Codepen example](https://codepen.io/team/Cloudinary/project/editor/AxrvPZ/)
-* [Plunker example](https://plnkr.co/edit/Tvrwxt2AW8K5DEhuTKhB?p=preview)
-* [Glitch example](https://glitch.com/edit/#!/cloudinary-angular-sdk)
-
-## Installation ######################################################################
-Install the SDK version that supports the Angular version you are using:
-
-* For Angular 2, use `2.x`
-* For Angular 4, use `4.x`
+## Version Support
 * For Angular versions greater than 5, use `5.x`
```

Comment:
Can you please clarify this with more words or add a table? I wouldn't understand what this means.

Reply:
updated

Reply:
Please use a table in this as the format states in our template, we're trying to provide the same look and feel for all SDKs.